requirements.txt

```diff
@@ -59,9 +59,11 @@ pyflakes==1.6.0
 Pygments==2.2.0
 pylint==1.9.2
 pyparsing==2.2.0
-pytest==3.9.3
+pytest==5.2.4
 pytest-cov==2.5.1
+pytest-forked==1.1.3
```

Comment:
These test have a bunch of shared external state — running forked tests will likely cause a lot of test failures.

Reply:
fwiw i pulled that out of pip-freeze - probably a dep of pytest-xdist

Reply:
also note in other PR: i wasnt aware of pytest-watch (that it existed or what it did), and pytest-xdist dupes looponfail. could remove one of these